srcpkgs/bluez/template

```diff
@@ -44,6 +44,8 @@ post_install() {
 	vsv bluetoothd
 
 	vdoc ${FILESDIR}/README.voidlinux
+
+	rm ${DESTDIR}/etc/bluetooth/mesh-main.conf
```

Comment:
Why is this removed?

Reply:
This file is removed because it should only be in the bluez-mesh package and packaging is failing otherwise with a conflict error.